### ISSUE TITLE
Uploading a feed document signature error

### ISSUE THREAD

**Issue by antdesp** [jlevers/selling-partner-api#12]
When uploading the document feed the createFeedDocument method works fine but when I try to upload the document I always get the message "The request signature we calculated does not match the signature you provided. Check your key and signature method."

Snippet
```
use SellingPartnerApi;

const CONTENT_TYPE = "text/xml"; 
$feedsApi = new Api\FeedsApi();
$createFeedDocSpec = new Model\Feeds\CreateFeedDocumentSpecification(["content_type" => CONTENT_TYPE]);
$feedDocumentInfo = $feedsApi->createFeedDocument($createFeedDocSpec);

$docToUpload = new Document($feedDocumentInfo->getPayload(), CONTENT_TYPE)
$docToUpload->upload($xml_document);
```

are there other permissions to be enabled in AWS?

**Comment by jlevers** [comment]
Sorry for taking a while to respond here. I'll look into this more in the next day or two.

**Comment by DydyDevelopper** [comment]
Same issue for me when trying to upload inventory feed. Can you please check and/or provide a real example ? Not clear in the doc. Thanks a lot for your help. Dylan

**Comment by jlevers** [comment]
This should be fixed in the latest version (2.0.6). I also updated the README feed document upload example to this:

```php
use SellingPartnerApi\Api\FeedsApi;
use SellingPartnerApi\Model\Feeds;

const CONTENT_TYPE = "text/xml";  // This will be different depending on your feed type
$feedsApi = new FeedsApi();
$createFeedDocSpec = new Feeds\CreateFeedDocumentSpecification(["content_type" => CONTENT_TYPE]);
$feedDocumentInfo = $feedsApi->createFeedDocument($createFeedDocSpec);

$documentContents = file_get_contents("<your/feed/file.xml>");

$docToUpload = new SellingPartnerApi\Document($feedDocumentInfo->getPayload(), CONTENT_TYPE);
$docToUpload->upload($documentContents);
```
